### DISCUSSION TITLE
Install from .ZIP

### DISCUSSION THREAD

**Discussion by Shinaryia** [Palakis/obs-websocket#871] [Q&A]
I'm most likely a noob and missing something but since there is no installer option I have to install it manually but I'm having difficulty with that. i have extracted the files and tried putting them in the plugins folder but obs doesn't seem to be registering that I put the files there.  
What am I doing wrong as there are no instructions on how to install this.

**Comment by ghost** [comment]


**Reply by tt2468** [reply]
Your issue is not related to the one asked by OP. Your issue is because you're trying to install OBS on a raspberry pi.

obs-websocket does not support the Raspberry Pi, and neither does OBS itself. Support for Raspberry Pi will not be provided.

**Comment by tt2468** [comment]
I'd need to know which version of obs-websocket and OBS Studio you have. Do you also have OBS installed into a custom path?

I highly recommend installing obs-websocket via our .exe installer if you are on Windows.

**Reply by Shinaryia** [reply]
Thank you for the reply, 
I don't see an install from .exe option
![image](https://user-images.githubusercontent.com/88183193/167762636-77f488ce-9d21-417e-8254-a99b2fce38b4.png)

Im using OBS 27.2.3 atm and far as i know i think i installed it in the default directory since the files are located on my os drive.

**Reply by tt2468** [reply]
That's where you download the source code, not the binary plugin files. See https://github.com/obsproject/obs-websocket/releases/tag/4.9.1 to download the .exe.
